### ISSUE TITLE
Blocked by Yahoo

### ISSUE THREAD

**Issue by AbhijitKulkarni95** [sendgrid/sendgrid-python#933]
Error: "421 4.7.0 [XXX] Messages from x.x.x.x temporarily deferred due to user complaints - 4.16.55.1" when sending email to Yahoo

How to resolve this issue?

**Comment by thinkingserious** [comment]
Hello @AbhijitKulkarni95,

Thanks for submitting a GitHub issue! We are very sorry that you are running into this problem. In order to better serve you, as this does not present itself as a library specific issue, we would like to ask that you reach out to our support team at <a href="https://support.sendgrid.com">https://support.sendgrid.com</a>.

Thank you!

SendGrid DX Team
